crates/boundless-market/src/contracts/artifacts/Account.sol

```diff
@@ -16,8 +16,7 @@ struct Account {
     uint96 balance;
     /// @dev Balance of staked tokens.
     uint96 stakeBalance;
-    /// @dev flags is a bitfield for account-wide flags. Currently only the least significant
-    /// bit is used to indicate that the account is frozen.
+    /// @dev flags is a bitfield for account-wide flags. Currently unused.
```

Comment:
Hmm shouldn't we roll it back into `requestFlagsInitial`?

Reply:
Interesting, I can't think of another use case for it, and once the system is truly permissionless you could always get around a flag by creating a new account. So I'm in favor of removing

Reply:
dropped with https://github.com/boundless-xyz/boundless/pull/359/commits/892d7dd7c22db67e78efdc2bf6b2e04067767014